docs/domain_with_free_ssl.md

```diff
@@ -1,6 +1,6 @@
 # Using Let's Encrypt with Zappa
 
-This guide will show you the slightly convoluted way of running a Zappa website on a custom domain with a free valid SSL certificate via Let's Encrypt.
+This guide will show you the slightly convoluted way of running a Zappa website on a API Gateway Custom Domain Names with a free valid SSL certificate via Let's Encrypt.
```

Comment:
Should probably be 'on an API Gateway Custom Domain Name' instead of 'on a API Gateway Custom Domain Names' -- `s/a/an`, `s/Names/Name`